packages/typespec-azure-portal-core/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from "./decorators.js";
+export * from "./keys.js";
+export { $lib } from "./lib.js";
+export * from "./types.js";
+export * from "./validate.js";
```

Comment:
```suggestion
export {$onValidate} from "./validate.js";
```


try to limit what you export as everything becomes public api for your package and change will break users



Double check for the other `export *` if you do actually want that

Reply:
ah! I see yes checked everything will be needed